src/xdocs/checks.xml

```diff
@@ -805,5 +799,26 @@
       </tr>
     </table>
     </section>
+
+    <section name="Non-Standard Checks">
```

Comment:
Technically it is still a check by inheritance. Test method uses `getCheckstyleChecks`. I think we will confuse ourselves if we say they are "Non-Checks", so I am leaning more to holders for now since all of them end with holder.